helix-core/src/main/java/org/apache/helix/manager/zk/ZKUtil.java

```diff
@@ -66,6 +67,10 @@ public static boolean isClusterSetup(String clusterName, String zkAddress) {
   }
 
   public static boolean isClusterSetup(String clusterName, HelixZkClient zkClient) {
```

Comment:
If we have `public static boolean isClusterSetup(String clusterName, RealmAwareZkClient zkClient)`, it seems keeping this old API `public static boolean isClusterSetup(String clusterName, HelixZkClient zkClient)` is redundant, because `HelixZkClient` is a subclass of `RealmAwareZkClient`?

Reply:
Yes. Will remove.